apps/explorer/src/ui/TransactionType.tsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    type ExecutionStatusType,
+    type TransactionKindName,
+} from '@mysten/sui.js';
+
+import { ReactComponent as ContentSuccessStatus } from '../assets/SVGIcons/12px/Check.svg';
+import { ReactComponent as ContentFailedStatus } from '../assets/SVGIcons/12px/X.svg';
+
+type TxStatus = {
```

Comment:
This should follow the pattern for all other ui components (named `TransactionTypeProps`), and should be exported.